### PR TITLE
Add pg8000 python example

### PR DESCRIPTION
**Pull request by bobbyiliev**
Adding `pg8000` example, as per the discussion here: https://github.com/MaterializeInc/materialize/pull/15888

### CODE REVIEW

**Review by morsapaes — CHANGES_REQUESTED** [review]
(👀 🚨)

Ah, interesting! I don't think I've ever seen anyone using `pg8000` in the wild. Something minor that shouldn't block merging this, but could be addressed as a follow-up: since we also have an example for `pyscopg3`, it'd be good to be consistent and use the approach and naming for each driver; right now, we have separate files for each operation for `pyscopg2`, all operations in one file for `pg8000`, and a single file for `psycopg3`.

**Review by morsapaes — APPROVED** [review]

File: python/pg8000-example.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import pg8000.native
+import ssl
+
+ssl_context = ssl.create_default_context()
+ssl_context.check_hostname = ssl.CERT_REQUIRED
```
Comment (aalexandrov) [inline python/pg8000-example.py:7]: as @vmarcos mentioned in MaterializeInc/materialize#15888, `ssl.CERT_REQUIRED` seems to be the default setting with `ssl.create_default_context()` according to [the `ssl` module docs](https://docs.python.org/3/library/ssl.html#ssl.CERT_REQUIRED). I changed my code as follows to reflect that:

```python
# verify_mode=ssl.CERT_REQUIRED is the default
ssl_context = ssl.create_default_context()
```

**Review by aalexandrov — APPROVED** [review]
Looks good, I have one remark (forwarding a nit from @vmarcos on my own PR)...

File: python/pg8000-example.py (hunk 1 above)
Comment (bobbyiliev) [reply python/pg8000-example.py]: That is a good point! Just updated this accordingly

**Comment by bobbyiliev** [comment]
> Ah, interesting! I don't think I've ever seen anyone using `pg8000` in the wild. Something minor that shouldn't block merging this, but could be addressed as a follow-up: since we also have an example for `pyscopg3`, it'd be good to be consistent and use the approach and naming for each driver; right now, we have separate files for each operation for `pyscopg2`, all operations in one file for `pg8000`, and a single file for `psycopg3`.

That is a good point @morsapaes! I've created a separate directory for the `pg8000` examples, that way we could keep things structured for each library. I will open a new PR to do the same thing for `pyscopg3`!

**Pull request merged**